climada/util/coordinates.py

```diff
@@ -502,106 +472,6 @@ def convert_wgs_to_utm(lon, lat):
     epsg_utm_base = 32601 + (0 if lat >= 0 else 100)
     return epsg_utm_base + (math.floor((lon + 180) / 6) % 60)
 
-def utm_zones(wgs_bounds):
-    """Get EPSG code and bounds of UTM zones covering specified region
-
-    Parameters
-    ----------
-    wgs_bounds : tuple
-        lon_min, lat_min, lon_max, lat_max
-
-    Returns
-    -------
-    zones : list of pairs (zone_epsg, zone_wgs_bounds)
-        EPSG code and bounding box in WGS coordinates.
-    """
-    lon_min, lat_min, lon_max, lat_max = wgs_bounds
-    lon_min, lon_max = max(-179.99, lon_min), min(179.99, lon_max)
-    utm_min, utm_max = [math.floor((l + 180) / 6) for l in [lon_min, lon_max]]
-    zones = []
-    for utm in range(utm_min, utm_max + 1):
-        epsg = 32601 + utm
-        bounds = (-180 + 6 * utm, 0, -180 + 6 * (utm + 1), 90)
-        if lat_max >= 0:
-            zones.append((epsg, bounds))
-        if lat_min < 0:
-            bounds = (bounds[0], -90, bounds[2], 0)
-            zones.append((epsg + 100, bounds))
-    return zones
-
-def dist_to_coast(coord_lat, lon=None, signed=False):
```

Comment:
Instead of just removing it, we can have this function be a wrapper around `dist_to_coast_nasa`. However you prefer...

Reply:
I wouldn't even make this function deprecated. We can just as well keep it as an alias (see my latest commit).

Reply:
Alright 👍  I got confused with the `set_dist_coast`, but that's actually a method of `Centroids`.